apps/src/feedbackBlocks.js

```diff
@@ -85,6 +86,14 @@ FeedbackBlocks.prototype.show = function() {
   }
 };
 
+FeedbackBlocks.prototype.hideDiv = function() {
+  this.div.className += " hidden";
+};
+
+FeedbackBlocks.prototype.revealDiv = function() {
+  this.div.className = this.div.className.replace(/\hidden\b/,'');
```

Comment:
What do the `\h` and `\b` do in this?

If it's safe to use jQuery in this context it might be worth using `$(this.div).removeClass('hidden');`

Alternatively there's [blockly's implementation](https://github.com/code-dot-org/code-dot-org/blob/c692117679fe0c8aa9284b4f7459f1e5b10700c2/blockly-core/core/utils/utils.js#L66-L66) but might be overkill since we're not dealing with an svg el't here. Or [YMNNJq's version](http://youmightnotneedjquery.com/#remove_class)